lib/lndclient/errors.ts

```diff
@@ -1,8 +1,11 @@
-const codesPrefix = require('../constants/errorCodesPrefix').LND;
+import errorCodesPrefix from '../constants/errorCodesPrefix';
 
-const errors = exports;
+const codesPrefix = errorCodesPrefix.LND;
+const errors : any = {};
 
 errors.LND_IS_DISABLED = {
   message: 'lnd is disabled',
-  code: codesPrefix.concat(1),
+  code: codesPrefix.concat('1'),
```

Comment:
Changing to import syntax disabled the ability to extract a property directly (LND).
I added line 3 for this matter.
Pay attention to line 8 --> now that TS recognize that `codesPrefix` is of type string, it cannot be concatenated by number 1. I changed it to string, make sure it doesn't affect any logic.